map/mwm_tests/CMakeLists.txt

```diff
@@ -1,10 +1,5 @@
 project(mwm_tests)
 
-add_compile_options(
-  "-Wall"
```

Comment:
Don't remove this option please

Reply:
This compile option is added by default in root ```omim/CMakeLists.txt:240```
```cmake
# Only options related to warnings should be placed here.
# Other options should be set before all add_subdirectory calls.
add_compile_options(
  "-Wall"
)
```

Reply:
ok